custom/icds_reports/utils/data_pull/Oct-2019.sql

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Hey, thanks writing all these. 
I think Generally quering at agg_level =5 is slow we query as less as possible. But I think here it is kind of needed because 'column=0' we dont roll up. In that case, I think combining all query and making one query will be faster to fetch rather than running similar query one after the other. 
for comined one you can use the query with join as some column might not be present in the agg_awc_monthly view.

Reply:
I am not sure how heavy these are going to be, so I will take your input on it from experience.
My effort was to keep the order of columns as needed in the final report so i can just merge them later and not need to reorder, so i have combined them if they are consecutive columns fetching columns from the same table or set of tables.
If its really expensive and I should combine them and then do the reorder, let me know.
I understand i can arrange the columns in the select query accordingly as well, but since at times things are fetched from different tables/combination of tables,  they are in different queries.